src/backend/config/deployments.py

```diff
@@ -26,39 +25,44 @@ class ModelDeploymentName(StrEnum):
     SingleContainer = "Single Container"
 
 
-use_community_features = bool(strtobool(os.getenv("USE_COMMUNITY_FEATURES", "false")))
-
+use_community_features = Settings().feature_flags.use_community_features
 
+# TODO names in the map below should not be the display names but ids
 ALL_MODEL_DEPLOYMENTS = {
     ModelDeploymentName.CoherePlatform: Deployment(
+        id="cohere_platform",
```

Comment:
Should we have an enum with the ids? It might be easier to find all existing tools when deciding which ones to enable

Reply:
Yeah, I added a TODO we need the key of the map to also be the ID not the display name